program/src/instruction.rs

```diff
@@ -40,18 +40,20 @@ pub enum TokenWrapInstruction {
     ///
     /// Accounts expected by this instruction:
     ///
-    /// 0. `[writeable]` Unwrapped token account to wrap
-    /// 1. `[writeable]` Escrow of unwrapped tokens, must be owned by:
+    /// 0. `[s]` Transfer authority on unwrapped token account
+    /// 1. `[w]` Escrow of unwrapped tokens, must be owned by:
+    /// 2. `[w]` Unwrapped token account to wrap
     ///    `get_wrapped_mint_authority(wrapped_mint_address)`
-    /// 2. `[]` Unwrapped token mint
-    /// 3. `[writeable]` Wrapped mint, must be initialized, address must be:
+    /// 3. `[w]` Recipient wrapped token account
+    /// 4. `[w]` Wrapped mint, must be initialized, address must be:
     ///    `get_wrapped_mint_address(unwrapped_mint_address,
     ///    wrapped_token_program_id)`
-    /// 4. `[writeable]` Recipient wrapped token account
     /// 5. `[]` SPL Token program for unwrapped mint
     /// 6. `[]` SPL Token program for wrapped mint
-    /// 7. `[signer]` Transfer authority on unwrapped token account
-    /// 8. `..8+M` `[signer]` (Optional) M multisig signers on unwrapped token
+    /// 7. `[]` Unwrapped token mint
+    /// 8. `[]` Wrapped mint authority, address must be:
+    ///    `get_wrapped_mint_authority(wrapped_mint)`
+    /// 9. `..8+M` `[s]` (Optional) M multisig signers on unwrapped token
```

Comment:
Re-ordered according to https://github.com/solana-program/token-wrap/pull/6#discussion_r1944951701